.github/workflows/install_test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python_version: ["3.8", "3.9", "3.10"]
+        python_version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
If we are going to support 3.11, shouldn't we also run all our unit tests with 3.11, not just the install test?